posthog/tasks/webhooks.py

```diff
@@ -3,12 +3,15 @@
 from typing import Tuple
 
 import requests
+import urllib3
 from celery import Task
 from django.conf import settings
 
 from posthog.celery import app
 from posthog.models import Action, Event, Team
 
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
```

Comment:
Q: Why is this needed?

Reply:
It's a webhooks thing, we have `verify=false` on the request to Slack/Teams/Discord/whatever and `urllib3` complained in logs, but it's not a substantial issue and only was spamming stdout